WooCommerce/Classes/ViewModels/CardPresentPayments/CardPresentModalSuccess.swift

```diff
@@ -10,6 +10,9 @@ final class CardPresentModalSuccess: CardPresentPaymentsModalViewModel {
     /// Closure to execute when secondary button is tapped
     private let emailReceiptAction: () -> Void
 
+    /// Closure to execute when auxiliary button is tapped.
+    private let backToOrdersAction: () -> Void
```

Comment:
Suggestion: let's rename this as noReceiptAction since Simple Payments will return them to the Orders screen, but In-Person Payments returns them to the originating Order.